### PR TITLE
Toggle menu

### PR DESCRIPTION
**Pull request by ChaabaneAnas**
add closebtn functionality
add transition

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,101 @@
...
-## Built With
+Please do the following **steps in this order**:
 
-- HTML & CSS
-- No Frameworks Used
-- No Technologies used
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
```
Comment (thecodechaser) [inline README.md]: - [ ] You changed your README file with this new README. It looks like it belongs to linters. Please make sure to change this README file with a professional one. I would suggest you copy the README file from your main branch and replace this file.

File: index.html [hunk 2]
```diff
@@ -12,12 +12,15 @@
    <header id="header">
        <h1><a href="index.html">Welcome</a></h1>
        <nav>
-         <ul id="nav">
+         <div class="nav">
+         <ul class="navL" >
             <li> <a href="#works">Porfolio</a></li>
             <li> <a href="#about-myself">about</a></li>
             <li> <a href="#contact">contact</a></li>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please use capitalization for `About` and `Contact` menu options.

File: index.html [hunk 3]
```diff
@@ -12,12 +12,15 @@
...
          </ul>
-       <img src="images/Enabled.png" alt="Enabled">
+         <a class="closebtn" id="closebtn">&times;</a>
```
Comment (thecodechaser) [inline index.html:21]: - [ ] Please reduce the weight of your close(X) button and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ChaabaneAnas, and @MaiRizk👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] I would suggest group members, commit equally to pair-programming projects.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @ChaabaneAnas,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by ChaabaneAnas** [comment]
Toggle menu aded

**Pull request merged**